graphistry/layout/networkx.py

```diff
@@ -0,0 +1,35 @@
+def to_networkx(g):
```

Comment:
Is this for reusing networkx methods, or a dev convenience for stuff like rendering?

Maybe it's just an internal method. From a public interface perspective, better if we do something like `sugiyuma` -> `graphistry plottable via nodes/edges dataframes` -> `networkx`

Reply:
Convenience to render something. The layout mechanics does not (intentional) depend on NetworkX. It would have been eaiser to rely on NetworkX.

Reply:
Removed the NetworkX references but it means that the demo notebook had to go as well.

Reply:
The NetworkX reference is gone in the layout module but used it in the demo notebook in order to visualize things. Having in the notebook only a bunch of coordinates is not really pleasing and I suppose it's alright to have NetworkX in the notebook.